Source/FieldSolver/SpectralSolver/SpectralAlgorithms/AvgGalileanAlgorithm.cpp

```diff
@@ -0,0 +1,371 @@
+#include "FieldSolver/SpectralSolver/SpectralAlgorithms/AvgGalileanAlgorithm.H"
+#include "Utils/WarpXConst.H"
+#include <cmath>
+
+using namespace amrex;
+
+/* \brief Initialize coefficients for the update equation */
+AvgGalileanAlgorithm::AvgGalileanAlgorithm(const SpectralKSpace& spectral_kspace,
+                         const DistributionMapping& dm,
+                         const int norder_x, const int norder_y,
+                         const int norder_z, const bool nodal,
+                         const amrex::Array<amrex::Real,3>& v_galilean,
+                         const Real dt)
```

Comment:
Here I think it would be better to add a space between the function name and the parentheses, following WarpX's style guidelines:
```suggestion
AvgGalileanAlgorithm::AvgGalileanAlgorithm (const SpectralKSpace& spectral_kspace,
                                            const DistributionMapping& dm,
                                            const int norder_x, const int norder_y,
                                            const int norder_z, const bool nodal,
                                            const amrex::Array<amrex::Real,3>& v_galilean,
                                            const Real dt)
```

Reply:
Agreed.